packages/browser/test/integration/common/init.js

```diff
@@ -51,7 +51,11 @@ function initSDK() {
       }
 
       // One of the tests use manually created breadcrumb without eventId and we want to let it through
-      if (breadcrumb.category.indexOf("sentry" === 0) && breadcrumb.event_id) {
+      if (
+        breadcrumb.category.indexOf("sentry" === 0) &&
```

Comment:
If we have `allowSentryBreadcrumbs`, maybe we can drop first two checks and reuse it everywhere?

Reply:
It's already the case ^^